Lead Backend Developer.md

```diff
@@ -0,0 +1,68 @@
+---
+---
+
+# Senior Backend Developer
+
+_Victoria BC &middot; Full-Time_
+
+Sendwithus is a YC W2014 backed company looking for an amazing Lead Backend Developer. We are a fun loving, hard working, and motivated team that loves to ship code and create amazing customer experiences. We want someone who can work with our team of developers and contribute to our lively team culture. We are a small start-up based in Victoria BC and San Francisco CA and we want that perfect person to grow and scale with us.
+
+<!-- more -->
+
+__DESCRIPTION__
+
+We're looking for a full stack engineer to help build new application features, scale our backend systems, and grow into a senior leader on our engineering team.
+As an early engineer, you'll be joining a rapidly growing startup where you will be a leader in scaling a product people love while building a billion dollar company.
+This is a high-impact, high-opportunity technical role. You'll be working with modern technologies to build a product that has a high throughput. You'll be talking with actual customers to create features that make a real difference.
```

Comment:
You'll be working with modern technologies to build a complex high throughput product.